src/include/sof/trace/trace.h

```diff
@@ -175,7 +175,7 @@ int trace_filter_update(const struct trace_filter *elem);
 	} log_entry = {						\
 		lvl,						\
 		comp_class,					\
-		params,						\
+		n_params,						\
```

Comment:
While it makes the macro readable, it does not belong to this commit as per the subject and message

Reply:
Thanks @ujfalusi for the very thorough review! Technically you're right: this is not just a  comment change. In practice this macro parameter name is used only once in the macro definition, must be a compile-time constant and the macro itself is used in only one place in the entire code base right now. So the risk of conflict and possible consequences are either zero or extremely close to it and that's why I chose to save one extra commit for these 4 characters. As you've noticed the rest of the PR is carefully divided in 9 different commits with hopefully good commit messages.

Now if you insist... :-)

Reply:
I'm fine - CI has already run. Lets merge.

Reply:
@marc-hb, @lrgirdwo, I did not wanted to insist either.

This is a really nice cleanup series!